WORKING_GROUPS.md

```diff
@@ -0,0 +1,173 @@
+# IPFS Project Working Groups
+
+## Index
+
+- [Description](#description)
+- [Working Group Lifecycle](#working-group-lifecycle)
+- [Current Working Groups](#current-working-groups)
+- [Other](#other)
+
+## Description
+
+The IPFS Project Working Groups are teams of people that are appointed to research, develop and deploy new work under the scope that the working group is focused. This structure is designed to provide clarity and direction to the project, enabling individual contributors to focus their time and energy on the areas they are most interested.
+
+Each Working Group has its own cadence, lifecycle and objectives.
+
+## Working Group Lifecycle
+
+TODO: Define how a working group gets created, evaled and how to decide if its purpuse has been fulfilled and should be dismantled
+
+## Current Working Groups
+
+### Release
```

Comment:
Release Working Groups 
- JavaScript Releases
- Go Releases

Reply:
I considered this and I'm confident that releasing JS and Go can be handled by the same team. In the end, the Release team should focus on auditing the release, ensuring that the quality bar didn't lower (check features added, CI, documentation and so on), that communication with the community and partners happens (changelog, post, twitter, reddit, emails and so on).

Yes, there are some people that have more context of what is happening in Go and others in JS, but in the end it is the same protocol and given that this is a Working Group and not a Working individual, we can have 2~4 people that share enough context to be capable of _launching the rocket_ with the same level of quality.